pkg/runtime/unstructured/BUILD

```diff
@@ -0,0 +1,36 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Don't forget package needs to move to apimachinery/runtime/unstructured and test case will need to be in pkg/api (the one that references the API types).